.github/workflows/ci.yml

```diff
@@ -15,10 +15,6 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@e2f20e631ae6d7dd3b768f56a5d2af784dd54791 # v2.5.0
-    - name: Setup JDK
-      uses: actions/setup-java@de1bb2b0c5634f0fc4438d7aa9944e68f9bf86cc # v3.6.0
-      with:
-        distribution: corretto
-        java-version: ${{ matrix.java }}
```

Comment:
This is a change in java version, it looks like: this used to run on both java 11 and 17, but now only runs on 21. That’s probably fine? I guess it means any consuming libraries will need to upgrade, but I _think_ that’s ok.

We should remove the matrix if only using a single java version, though, and update [the release flag](https://github.com/guardian/atom-maker/blob/5a52d1ea06d18db0d8bdcbf55bc7bff6e0ae59d4/build.sbt#L11) to version 21.

Alternatively, I suppose we could stick to 11?

Reply:
Oh this was my misunderstanding, as i saw that there was already a `.tools-versions` pointing to 21.  So if I added 11 as a new line in `.tools-versions` would that work, or am I being overly simplistic?

Reply:
I was imagining replacing the 21 version with an 11 version, but I hadn’t thought about whether something already needs version 21 – hmmmm. I don’t know what it means if there are multiple version lines in `.tools-versions` (will google), but I doubt it has the same effect as the current matrix.

Where is this library used? If we could publish a snapshot release built using java 21 and test it with the consumers of this library I’d feel pretty happy about moving to 21 and dropping 11 and 17 altogether.

Reply:
[You can specify multiple versions](https://asdf-vm.com/manage/configuration.html#tool-versions) (on the same line instead of different lines), but they’re interpreted as preferences: “this version, falling back to this version”.

Reply:
I guess our other option is to use `setup-sbt` directly, and stick to having the multiple version situation we already have? That might be the safest/easiest choice, actually.

Reply:
I agree - that would be the better way to do it.  Thanks for flagging this!

Reply:
No problem! Looking at maven central, [the only known dependent for atom-publisher-lib is atom-manager-play](https://central.sonatype.com/artifact/com.gu/atom-publisher-lib_2.13/dependents) (i.e., the other library in this repo) – so we might be fine to update to Java 21?

Still, sticking to the current versions sounds most straightforward, and we can come back to the version update separately.

Reply:
Ah, a couple of our apps depend on that: https://github.com/search?q=org%3Aguardian%20atom-manager-play&type=code

Reply:
I've opted for

```
- name: Setup sbt
      uses: sbt/setup-sbt@v1.1.0
```
      
instead of pinning the version, as that's consistent with how `setup-scala` references it